install-panel.sh

```diff
@@ -604,7 +596,7 @@ function debian_based_letsencrypt {
   systemctl stop nginx
 
   FAILED=false
-  certbot certonly --standalone -d "$FQDN" || FAILED=true
+  certbot certonly --email "$email" --standalone -d "$FQDN" || FAILED=true # -q could be added because it already checks if it failed
```

Comment:
Good use of `--email`, I agree then you can use `--no-eff-email`. Do it! :+1: Still think the user should accept the TOS since it is a legal document.